carbon-design-system/carbon
# fix: Have JAWS to announce 'Readonly' status on NumberInput

Closes https://github.com/carbon-design-system/carbon/issues/16530

This PR fixes the issue that JAWS was Not Announcing 'Readonly' Property for Input

#### Changelog

**New**

- Adds  `aria-readonly` to `<Input/> `

#### Testing / Reviewing

- Run JAWS on windows machine
- go to `NumberInput` story and update the `readOnly` prop to `true` from playground 
- Verify that JAWS announces readOnly status on the input 
- Verify with both Tab and arrow-key navigations
